src/modules/crud.js

```diff
@@ -0,0 +1,177 @@
+<i class="fa-solid fa-trash-can"></i>
+</span>
+`;
+  listOfTasks.appendChild(newTask);
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((checkboxInput) => {
+    checkboxInput.addEventListener('click', () => {
+      checkboxInput.nextElementSibling.classList.toggle('strike');
+      checkboxInput.parentNode.classList.toggle('clicked-on');
+      const ellipsisIcon = checkboxInput.parentNode.childNodes[5];
+      ellipsisIcon.classList.toggle('inactive-ellipsis');
+      const trashIcon = checkboxInput.parentNode.childNodes[7].childNodes[1];
+      trashIcon.classList.toggle('active-trash');
+      completeItem();
+    });
+  });
+  // Create and send new task to local storage
+  const newListItem = new Task(newDescription, false, checkbox.length);
+  taskArray.push(newListItem);
+  const stringedItems = JSON.stringify(taskArray);
+  localStorage.setItem('items', stringedItems);
+  // Edit task
+  const editItem = document.querySelectorAll('.ellipsis');
+  editItem.forEach((item) => {
+    item.addEventListener('click', () => {
+      item.parentNode.classList.add('clicked-on');
+      editItems(item.previousElementSibling);
+    });
+  });
+  // Remove tasks
+  const removeItem = document.querySelectorAll('.trashcan');
+  removeItem.forEach((item) => {
+    item.addEventListener('click', () => {
+      removeItems(item.parentNode);
+    });
+  });
+};
+
+// Clear all tasks
+const clearAllItems = () => {
+  clearAll.addEventListener('click', () => {
+    const localItems = localStorage.getItem('items');
+    const parsedData = JSON.parse(localItems);
+    const checkDelete = parsedData.filter((item) => item.completed === false);
+    for (let i = 0; i < checkDelete.length; i += 1) {
+      checkDelete[i].index = i + 1;
+    }
+    localStorage.setItem('items', JSON.stringify(checkDelete));
+    window.location.reload();
+  });
+};
+
+// Get data from local storage changes
+const getItemsLocal = () => {
+  const localItems = localStorage.getItem('items');
+  const items = JSON.parse(localItems);
+  items.map((item) => {
+    taskArray.push(item);
+    const newItemList = document.createElement('div');
+    newItemList.classList.add('new-task');
+    const newId = item.index;
+    newItemList.setAttribute('id', newId);
+    newItemList.innerHTML = `
+  <input type="checkbox" class="checkbox" id="${item.index}">
+  <span class="eachItem" id="span${item.index}">${item.description}</span>
+  <span id="ellipsis${item.index}" class="ellipsis">
+  <i class="fa-solid fa-ellipsis-vertical"></i>
+  </span>
+  <span class="trashcan">
+  <i id="trash${item.index}" class="fa-solid fa-trash-can"></i>
+  </span>
+  `;
+    listOfTasks.appendChild(newItemList);
+    const editItem = document.querySelectorAll('.ellipsis');
+    editItem.forEach((item) => {
+      item.addEventListener('click', () => {
+        item.parentNode.classList.add('clicked-on');
+        editItems(item.previousElementSibling, newItemList);
```

Comment:
You are repeating the same code, I suggest to try creating a function and then reuse the code